libcontainer/init_linux.go

```diff
@@ -99,6 +97,26 @@ func StartInitialization() (retErr error) {
 		}
 	}()
 
+	// Set up logging. This is used rarely, and mostly for init debugging.
+	logLevel := int(logrus.DebugLevel) // default to debug
+	// Passing log level is optional; currently libcontainer/integration does not do it.
+	if levelStr := os.Getenv("_LIBCONTAINER_LOGLEVEL"); levelStr != "" {
+		logLevel, err = strconv.Atoi(levelStr)
+		if err != nil {
+			return fmt.Errorf("unable to convert _LIBCONTAINER_LOGLEVEL: %w", err)
+		}
+	}
```

Comment:
It looks like now we're defaulting to "Debug" logs, which previously wasn't the case? Should we only set the log-level if any was passed?

```suggestion
	// Passing log level is optional; currently libcontainer/integration does not do it.
	if levelStr := os.Getenv("_LIBCONTAINER_LOGLEVEL"); levelStr != "" {
		logLevel, err := strconv.Atoi(levelStr)
		if err != nil {
			return fmt.Errorf("unable to convert _LIBCONTAINER_LOGLEVEL: %w", err)
		}
		logrus.SetLevel(logrus.Level(logLevel))
	}
```

Reply:
done